publish.gradle

```diff
@@ -0,0 +1,56 @@
+// Configuration for Maven Central by Sonatype.
+// See: https://github.com/gradle-nexus/publish-plugin
+nexusPublishing {
+    repositories {
+        sonatype {
+            nexusUrl.set(uri('https://s01.oss.sonatype.org/service/local/'))
+            snapshotRepositoryUrl.set(uri('https://s01.oss.sonatype.org/content/repositories/snapshots/'))
+            username = System.getenv('MAVEN_USERNAME')
+            password = System.getenv('MAVEN_PASSWORD')
+        }
+    }
+}
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            from components.java
+
+            pom {
+                group = 'dev.openfga'
+                name = 'openfga-sdk'
+                version = '0.1.0'
+                description = 'This is an autogenerated Java SDK for OpenFGA. It provides a wrapper around the [OpenFGA API definition](https://openfga.dev/api).'
+                url = 'https://openfga.dev'
+                licenses {
+                    license {
+                        name = 'Apache-2.0'
+                        url = 'https://github.com/openfga/openfga/blob/main/LICENSE'
+                    }
+                }
+                developers {
+                    developer {
+                        name = 'OpenFGA'
+                        url = 'https://openfga.dev'
+                    }
+                }
+                contributors {
```

Comment:
Can you remove this section? We generally do not do this and people can get it from the commit history.

Otherwise, we'll have to update this every time someone contributes.